Java-Introduction.md

```diff
@@ -0,0 +1,123 @@
+# Introduction
+
+## What is Java?
+
+Java is a programming language developed by Sun Microsystems in 1995, which got later acquired by Oracle. It is now a full platform with lots of standard API's, open source API's, tools, huge developer community and is used to build the most trusted enterprise solutions by big and small companies alike. Android operating system is built fully with Java and its ecosystem. To know more about Java, read [this](https://java.com/en/download/faq/whatis_java.xml) and [this](http://tutorials.jenkov.com/java/what-is-java.html).
```

Comment:
Java is trademarked by Oracle. What I am saying is, whenever any keyword appears the first time (like `Java` in this article), link it to the official website that licenses it.

So, link the term `Java` to Oracle's Java homepage. And link Oracle to Oracle's homepage. Just trying to avoid a CDMA takedown.